arbigent-cli/src/main/kotlin/ReplayCommand.kt

```
@@ -0,0 +1,135 @@
+    help = "Hardware UDID selecting a specific physical iPhone.",
+  )
+
+  internal val iosRealDevicePort by defaultOption(
+    "--ios-real-device-port",
+    help = "Local port used to reach the XCTest runner on a physical iPhone.",
+  )
+
+  override fun run() {
+    val log = try {
+      ArbigentReplayLog.read(File(logPath))
+    } catch (exception: ArbigentReplayLogException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)